group-generators/generators/proof-of-humanity/utils.ts

```diff
@@ -0,0 +1,61 @@
+import readline from "readline";
+import { ethers } from "ethers";
+import { JsonRpcProvider } from "@group-generators/helpers/data-providers/json-rpc";
+import { FetchedData } from "topics/group";
+
+const isRegistred = async (address: string): Promise<FetchedData> => {
+  const pohContractAddress = "0xC5E9dDebb09Cd64DfaCab4011A0D5cEDaf7c9BDb"
+  const jsonRPCProvider = new JsonRpcProvider(process.env.JSON_RPC_URL);
+  const getRegistrationABI =["function isRegistered(address _submissionID) external view returns (bool)"];
+  const pohContract = new ethers.Contract(pohContractAddress, getRegistrationABI, jsonRPCProvider);
+  const isRegistered = pohContract.isRegistered(address)
+  return isRegistered
+};
+
+export const areRegistred = async (addresses: FetchedData, max: number): Promise<number> => {
```

Comment:
Maybe add a comment explaining that these functions are helpful to know how many accounts are effectively registered? It was used to have some statistics to claim if. the data was correct by fetching the endpoint